test/t9010-gitignore-updated-generated-to-normal.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/sh -e
+
```

Comment:
Please add the copyright blurb (copy/paste from another test case) with "Copyright (C) 2024  [Your preferred name / email]"